recipes/plugins_example.ipynb

```diff
@@ -7,17 +7,17 @@
    "source": [
```

Comment:
<p>there are several locations in the sentences with 2 spaces instead of one</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
You mean the descriptive markdown cells? I ran a quick check and haven't seen any. But I'm checking in browser so it's not super-easy to do. Can you point me to an example?

Reply:
never mind, this is only minor and I am not sure if this is on the notebook or on the review app